app/gui2/src/components/SvgButton.vue

```diff
@@ -1,47 +1,30 @@
 <script setup lang="ts">
+import MenuButton from '@/components/MenuButton.vue'
 import SvgIcon from '@/components/SvgIcon.vue'
 import type { URLString } from '@/util/data/urlString'
 import type { Icon } from '@/util/iconName'
 
 const _props = defineProps<{
   name: Icon | URLString
+  label?: string
   disabled?: boolean
 }>()
 </script>
 
 <template>
-  <button class="SvgButton button" :class="{ disabled }" :disabled="disabled ?? false">
-    <SvgIcon :name="name" draggable="false" />
-  </button>
+  <MenuButton :disabled="disabled" class="SvgButton">
+    <SvgIcon :name="name" />
+    <div v-if="label" v-text="label" />
```

Comment:
This could be a slot instead, so we can do more than a text.
```suggestion
    <slot name="label" />
```

Reply:
The concision of an attribute is nice in the next PR that uses a lot of `SvgButton`s; `MenuButton` can be used for anything outside the common pattern.